alfresco-search/src/test/java/org/alfresco/solr/content/SolrContentStoreTest.java

```diff
@@ -36,31 +36,87 @@
 import org.mockito.Mockito;
 import org.mockito.runners.MockitoJUnitRunner;

```

Comment:
We could have updated the deprecated class with org.mockito.junit.MockitoJUnitRunner :)

Reply:
Done!